torch/nn/utils/rnn.py

```diff
@@ -85,6 +85,14 @@ def byte(self):
         r"""Returns copy with `self.data` cast to byte type"""
         return type(self)(self.data.byte(), self.batch_sizes)
 
+    def to(self, *args, **kwargs):
+        r"""Performs dtype and/or device conversion on `self.data`"""
```

Comment:
Could you at least mention that this is similar to `` :meth:`torch.Tensor.to` ``?